src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -337,6 +338,7 @@ public SnowflakeStreamingIngestChannelInternal<?> openChannel(OpenChannelRequest
       payload.put("schema", request.getSchemaName());
       payload.put("write_mode", Constants.WriteMode.CLOUD_STORAGE.name());
       payload.put("role", this.role);
+      payload.put("is_iceberg_client", isIcebergMode);
```

Comment:
I believe Toby asked for a different name on the contract, can you pl check my pr? Can find it via go/prby/hmadan in chrome.
Also note that drop channel needs this property for iceberg clients.